src/wallet/rpc/encrypt.cpp

```diff
@@ -229,11 +226,10 @@ RPCHelpMan encryptwallet()
         throw JSONRPCError(RPC_WALLET_WRONG_ENC_STATE, "Error: running with an encrypted wallet, but encryptwallet was called.");
     }
 
-    // TODO: get rid of this .c_str() by implementing SecureString::operator=(std::string)
-    // Alternately, find a way to make request.params[0] mlock()'d to begin with.
+    // Consider finding a way to make request.params[0] mlock()ed
```

Comment:
Instead of repeating this in all places where SecureString is used, what about moving it to src/support/allocators/secure.h once?